store/migrate_test.go

```diff
@@ -185,6 +230,13 @@ type ACIInfoV0_2 struct {
 	Latest     bool
 }
 
+type ACIInfoV3 struct {
+	BlobKey    string
+	Name       string
+	ImportTime time.Time
+	Latest     bool
+}
+
```

Comment:
Actually that raises the point that since ACIInfo is no longer changing I can just scrap this struct entirely.